tests/go.mod

```diff
@@ -7,14 +7,12 @@ require (
 	github.com/elazarl/goproxy v0.0.0-20190421051319-9d40249d3c2f // indirect
 	github.com/elazarl/goproxy/ext v0.0.0-20190421051319-9d40249d3c2f // indirect
 	github.com/jmcvetta/randutil v0.0.0-20150817122601-2bb1b664bcff
+	github.com/kudobuilder/kudo v0.7.3
```

Comment:
lets not add this dependency also, we are using `go mod edit -require github.com/kudobuilder/kudo@${DS_KUDO_VERSION}` to use the correct version of kudo library in `run.sh`